hooks/tk-3dsmaxplus_actions.py

```diff
@@ -169,14 +169,14 @@ def _import_alembic(self, path):
             )
         )
     
-    def _merge(self, path, sg_publish_data):
+    def _merge(self, path_utf8, sg_publish_data):
         """
         Merge contents of the given file into the scene.
         
         :param path: Path to file.
         :param sg_publish_data: Shotgun data dictionary with all the standard publish fields.
         """
-
+        path = path_utf8.decode('utf-8')
```

Comment:
please use double quotes for strings in the tk code base.

Reply:
since unicode is so tricky, let's make sure we add plenty of comments to explain what is going on. I'll just comment for 3dsmax in the pull request but please have a look at what's appropriate for each DCC below.

Please add a comment explaining why we are doing this and what's going on. I suggest something like this:

```
# toolkit uses utf-8 encoded strings internally and the 3dsmax API expects unicode
# so convert the path to ensure filenames containing complex characters are supported
```